_pytest/assertion/__init__.py

```diff
@@ -113,7 +115,8 @@ def callbinrepr(op, left, right):
         for new_expl in hook_result:
             if new_expl:
                 if (sum(len(p) for p in new_expl[1:]) > 80*8
-                        and item.config.option.verbose < 2):
```

Comment:
I don't get what you mean with "before fixing the rewrite mod changes".

And I have to disagree - `callbinrepr` doesn't seem very long as-is, and moving the stuff under `if new_expl` to a new function (if that's what you propose) doesn't really change much, IMHO.